addons/im_livechat/static/src/embed/common/thread_model_patch.js

```diff
@@ -159,14 +159,15 @@ patch(Thread.prototype, {
         );
     },
 
+    get hasLivechatEnded() {
+        return (super.hasLivechatEnded && !this.chatbot) || this.chatbot?.completed;
+    },
```

Comment:
I don't understand this condition, the only necessary condition for "ended" should be `livechat_active` normally?